packages/libdrm/libdrm.spec

```
@@ -0,0 +1,63 @@
+Name: %{_cross_os}libdrm
+Version: 2.4.129
+Release: 1%{?dist}
+Summary: Direct Rendering Manager runtime library
```

For consistency with the devel package summary:
```suggestion
Summary: Direct rendering manager library
```